src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &-star {
+    height: 12px;
+    width: 12px;
+    background-size: cover;
+
+    margin: 2px;
+    margin-right: 6px;
+
+    background-image: url(/src/images/star.svg);
+  }
+
+  &--4 :nth-child(-n + 4) {
```

Comment:
There is an extra space before the pseudo-class ':nth-child'. It should be '&--4:nth-child(-n + 4)' without the space.